### PR TITLE
espeak-ng: inclusion of homepage

### PR DESCRIPTION
**Pull request by androidnisse**
**Summary**
- Inclusion of "homepage" to package.yml
- Part of [#411](https://github.com/getsolus/packages/issues/411)
- Added Apache 2.0 license to package.yml

**Test plan**
- Installed and played some text

**Checklist**
- [X] Package was built and tested against unstable

### CODE REVIEW

File: packages/e/espeak-ng/package.yml [hunk 1]
```diff
@@ -1,11 +1,13 @@
 name       : espeak-ng
 version    : '1.51'
-release    : 6
+release    : 7
```
Comment (joebonrichie) [inline packages/e/espeak-ng/package.yml:3]: nit: extra space here

File: packages/e/espeak-ng/package.yml [hunk 2]
```diff
@@ -1,11 +1,13 @@
...
 version    : '1.51'
-release    : 6
+release    : 7 
 source     :
     - https://github.com/espeak-ng/espeak-ng/archive/refs/tags/1.51.tar.gz : f0e028f695a8241c4fa90df7a8c8c5d68dcadbdbc91e758a97e594bbb0a3bdbf
+homepage   : https://github.com/espeak-ng/espeak-ng
 license    :
     - GPL-3.0
     - BSD-2-Clause
+    - Apache-2.0
```
Comment (joebonrichie) [inline packages/e/espeak-ng/package.yml:10]: nit: alphabetized licences

**Review by joebonrichie — CHANGES_REQUESTED** [review]
A couple of small nits to address

**Comment by androidnisse** [comment]
@joebonrichie should look better now.

File: packages/e/espeak-ng/package.yml [hunk 3]
```diff
@@ -1,11 +1,13 @@
...
-release    : 6
+release    : 7
 source     :
     - https://github.com/espeak-ng/espeak-ng/archive/refs/tags/1.51.tar.gz : f0e028f695a8241c4fa90df7a8c8c5d68dcadbdbc91e758a97e594bbb0a3bdbf
+homepage   : https://github.com/espeak-ng/espeak-ng
 license    :
-    - GPL-3.0
+    - Apache-2.0
     - BSD-2-Clause
+    - GPL-3.0
```
Comment (joebonrichie) [inline packages/e/espeak-ng/package.yml:10]: Oh sorry i didn't notice this before but `GPL-3.0` is a deprecated SPDX licence identifier. Should be either `GPL-3.0-or-later` or `GPL-3.0-only`.

From https://github.com/espeak-ng/espeak-ng/blob/master/src/espeak-ng.c#L6 it looks like it's `GPL-3.0-or-later`

**Review by joebonrichie — CHANGES_REQUESTED** [review]
One more thing i missed sorry

**Comment by androidnisse** [comment]
@joebonrichie hopefully everything is fixed now.

**Review by joebonrichie — APPROVED** [review]

**Pull request merged**